.circleci/config.yml

```diff
@@ -1943,3 +1967,16 @@ workflows:
           context:
             - oplabs-gcr
             - slack
+
+  scheduled-bindgen:
```

Comment:
Why is this scheduled instead of on every relevant PR / on develop?

Reply:
These tests make request to Etherscan and RPCs. If either of the APIs is not functioning then these tests would block PRs from being merged. Additionally, if external contributors open PRs, they don't have access to the Circl Ci context which holds the API keys